README.md

```diff
@@ -22,12 +22,7 @@ reading through the [WHATWG FAQ](https://whatwg.org/faq) if you are new here.
 Please add your name to the Acknowledgments section in your first pull request, even for trivial
 fixes. The names are sorted lexicographically.
 
-If you want to preview the spec locally, you can either use a locally installed copy of
-[Bikeshed](https://github.com/tabatkins/bikeshed) by running `make` or use the HTTP API version by
-running `make remote`.
-
-If you want to do a complete "local deploy" including commit and/or branch snapshots, run
-`make deploy`.
+If you want to verify your changes before submitting them to GitHub, run `make deploy`.
```

Comment:
I feel not great about recommending people do full `make deploy` with its three curlbikesheds; I'd rather continue recommending `make remote` as a quicker flow for local iteration.